src/contracts/BuildersManager.sol

```diff
@@ -21,6 +20,9 @@ contract BuildersManager is EIP712Upgradeable, Ownable2StepUpgradeable, IBuilder
   /// @inheritdoc IBuildersManager
   bytes32 public projectSchema;
 
+  /// @notice Multiplier for fixed-point arithmetic
+  uint256 internal _multiplier;
```

Comment:
is this contract already deployed and needing to be upgraded? because i don't think this is "upgrade safe". would need to add this at the end of the storage variable declarations.